checker/examples/MavenExample/pom.xml

```diff
@@ -15,7 +15,10 @@
     <!-- These properties will be set by the Maven Dependency plugin -->
     <!-- Change to jdk7 if using Java 7 -->
```

Comment:
I agree.  Based on my limited Maven knowledge, I think we would want to use separate profiles that would be activated based on the version of Java used.  Because I don't know Maven well, this would be several hours of work.  We would also need to duplicate the example source file and put the annotations in comments.  I propose not doing this right now.